openshift/minishift-addons/rhche-prerequisites/rhche-prerequisites.addon

```diff
@@ -12,5 +12,8 @@ oc adm policy add-role-to-user admin developer -n #{NAMESPACE}
 echo [CHE] Deploying Postgres
 oc new-app -f templates/postgres-template.yaml -n #{NAMESPACE}
 
+echo [CHE] Adding Minishift certificate as a secret
+ssh docker exec origin /bin/bash -c 'oc -n rhche secret new-basicauth openshift-identity-provider --username='-' --password='-' --ca-cert=./openshift.local.config/master/ca.crt'
```

Comment:
Well, not really since I need to have access to the Minishfit certificate. Doing it with `oc secret` would require me to have access to this certificate in some way this is not fully supported by the limited DSL of Minishift addons.